src/main/java/fiuba/tpp/reactorapp/controller/AdsorbateController.java

```diff
@@ -64,31 +64,31 @@ public void deleteAdsorbato(@PathVariable Long id){
     }
 
     @GetMapping(value = "")
-    public List<AdsorbatoResponse> getAdsorbatos(){
-        List<AdsorbatoResponse> adsorbatos = new ArrayList<>();
-        for (Adsorbato adsorbato: adsorbatoService.getAll()) {
-            adsorbatos.add(new AdsorbatoResponse(adsorbato));
+    public List<AdsorbateResponse> getAdsorbates(){
+        List<AdsorbateResponse> adsorbates = new ArrayList<>();
+        for (Adsorbate adsorbate : adsorbateService.getAll()) {
+            adsorbates.add(new AdsorbateResponse(adsorbate));
         }
-        return adsorbatos;
+        return adsorbates;
     }
 
     @GetMapping(value = "/buscar")
-    public List<AdsorbatoResponse> searchAdsorbatos(@RequestParam(required = false) String nombre, @RequestParam(required = false) Integer cargaIon){
-        List<AdsorbatoResponse> adsorbatos = new ArrayList<>();
-        AdsorbatoFilter filter = new AdsorbatoFilter(nombre,cargaIon);
-        for (Adsorbato adsorbato: adsorbatoService.search(filter)) {
-            adsorbatos.add(new AdsorbatoResponse(adsorbato));
+    public List<AdsorbateResponse> searchAdsorbates(@RequestParam(required = false) String nombre, @RequestParam(required = false) Integer cargaIon){
```

Comment:
jodido que queden las variables en español, no? parece que RequestParam tiene una opcion para bindear con otros nombres
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/bind/annotation/RequestParam.html

Reply:
Ahi agregue el parametro a la annotation

Reply:
piola. esto se repite en varios lugares igual no? o solo en donde marque

Reply:
Params hay en los dos search y ya los cambie a los dos, despues no pasa por que siempre use constructores por copia, asi que we are safe. Los campos en la base de cada entity quedaron en español. me parecem as fiel poner cargaIon que ionicCharge, igual si estan todos en desacuerdo se migra eso tambien